### DISCUSSION TITLE
Task exception was never retrieved

### DISCUSSION THREAD

**Discussion by jamietjx** [osrf/rmf_demos#220] [Q&A]
After following the installation and trying to launch the office demo, gazebo and rviz failed to start as i have received this issue on task exception was never retrieved in the terminal. 
![Screenshot from 2021-04-08 12-46-07](https://user-images.githubusercontent.com/77659587/113970213-15d1ef80-9869-11eb-8573-ecafc691b100.png)

**Comment by Yadunund** [comment]
Hi @jamietjx ,

We are in the process of moving our RMF codebase to a dedicated `open-rmf` organization. Several packages were renamed but the launch files in this repo have not been updated. Please follow the instructions [here](https://github.com/open-rmf/rmf) to setup your workspace. Apologise for the confusion. We will soon be updating the README in this repo to point users to our new org.

**Comment by jamietjx** [comment]
Hi Yadu,

Thank you for the information that you have provided.


Regards,
Jamie
________________________________
